app/main.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+#   encoding: UTF-8
+
+import logging
+import os.path
+import sys
+
+from sdc.rabbit import MessageConsumer
+from sdc.rabbit import QueuePublisher
+
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
+
+from app.response_processor import ResponseProcessor
+import app.settings
+
+
+def run():
+    logging.basicConfig(format=app.settings.LOGGING_FORMAT)
```

Comment:
Same here - do `from . import settings` and access settings directly rather than doing path manipulation